### PR TITLE
 Update Kiwi building documentation

### PR DESCRIPTION
**Pull request by aaannz**
- RPM certificate package info
- accepted sources format and examples
- source example
- troubleshooting and limitations
- new screenshot for profile creation

### CODE REVIEW

File: adoc/advanced_topics_image_management.adoc [hunk 1]
```diff
@@ -405,9 +422,34 @@ From the {susemgr}{webui} perform the following steps to configure a build host.
...
+    ...
+    <package name="rhn-org-trusted-ssl-cert-osimage" bootinclude="true"/>
+  </packages>
+...
+----
+
+====
 
 [[at.images.kiwi.buildchannels]]
-==== Define Container Build Channels with an Activation Key
```
Comment (mbologna) [inline adoc/advanced_topics_image_management.adoc]: Please state in a NOTE that an AK is mandatory for OS image builds

File: adoc/advanced_topics_image_management.adoc [hunk 2]
```diff
@@ -474,11 +557,63 @@ Kiwi source consists of at least `config.xml` file, usually `config.sh` and `ima
 to be implanted in final image under `root` subdirectory.
 
 For all information regarding Kiwi build system please see https://doc.opensuse.org/projects/kiwi/doc/[Kiwi documentation]
-[TODO]
+
+[NOTE]
 ====
-!!!!!!!!!!!! add link to published JeOS sources
+{suse} provides fex examples of fully functional image sources at https://github.com/SUSE/manager-build-profiles[SUSE/manager-build-profiles] public GitHub repository.
```
Comment (mbologna) [inline adoc/advanced_topics_image_management.adoc]: fex?

File: adoc/advanced_topics_image_management.adoc [hunk 3]
```diff
@@ -474,11 +557,63 @@ Kiwi source consists of at least `config.xml` file, usually `config.sh` and `ima
...
+        <package name="patterns-sles-Minimal"/>
+        <package name="aaa_base-extras"/> <!-- wouldn't be SUSE without that ;-) -->
+        <package name="kernel-default"/>
+        <package name="salt-minion"/>
+        ...
+    </packages>
+    <packages type="bootstrap">
+        ...
+        <package name="sles-release"/>
+        <package name="rhn-org-trusted-ssl-certosimage" bootinclude="true"/>
```
Comment (mbologna) [inline adoc/advanced_topics_image_management.adoc]: As an additional repeat for the end user, please also add a comment here saying that this is required

File: adoc/advanced_topics_image_management.adoc [hunk 4]
```diff
@@ -511,9 +646,22 @@ When you have selected a build profile detailed information about the selected p
...
+
+
+[[at.images.kiwi.limitations]]
+=== Limitations
+
+
 The following are some known pitfalls when working with images.
 
-* HTTPS certificates to access the registry or the git repositories should be deployed to the minion by a custom state file.
+* HTTPS certificates to access the HTTP sources or Git repositories should be deployed to the minion by a custom state file.
```
Comment (mbologna) [inline adoc/advanced_topics_image_management.adoc]: or configured directly on the minion by the end user

**Comment by aaannz** [comment]
Addressed all comments + run spellcheck on advanced topics file.

**Comment by mbologna** [comment]
Please make sure to address the explanation of the certificate generation:
you can see some details here: https://github.com/SUSE/spacewalk/blob/Manager-Kiwi-OS-image-building/spacewalk/certs-tools/mgr-package-rpm-certificate-osimage.sgml

Basically: 
- on new installations: RPM package created by default
- on upgrades: the script is called upon upgrade of `spacewalk-certs-tools` package, and it tries to create the package using the certificate looking for it in the default directory `/root/ssl-build`. If the location is not available or the user has changed the directory/certificate, the user MUST run the script manually by passing the full path of the certificate as an option.

**Comment by aaannz** [comment]
Added info about new installation and upgrade and its relation to certificate rpm. Also added a note in trougleshooting about certificate rpm and link back to build host creation.

**Comment by keichwa** [comment]
Next time, we'll consolidate the usage of btn: and menu: and other asoc constructs.

**Pull request merged**